pkg/webhook/resources/setting/validator_test.go

```diff
@@ -218,3 +218,86 @@ func Test_validateSSLProtocols(t *testing.T) {
 		})
 	}
 }
+
+func Test_validateNTPServers(t *testing.T) {
+	testCases := []struct {
+		name        string
+		args        *v1beta1.Setting
+		expectedErr string
+	}{
+		{
+			name: "valid empty ntp servers",
+			args: &v1beta1.Setting{
+				ObjectMeta: v1.ObjectMeta{Name: settings.NTPServersSettingName},
+				Value:      "",
```

Comment:
Not sure if allowing empty NTP Servers is ok or not. 
If we set empty NTP Servers. `systemd-timesyncd.service` fallback to `0.suse.pool.ntp.org`

```bash
harvester-node-0:~ # cat /etc/systemd/timesyncd.conf

[Time]
NTP =
```

```bash
harvester-node-0:~ # systemctl status systemd-timesyncd.service
● systemd-timesyncd.service - Network Time Synchronization
     Loaded: loaded (/usr/lib/systemd/system/systemd-timesyncd.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2024-05-27 09:44:47 UTC; 22s ago
       Docs: man:systemd-timesyncd.service(8)
   Main PID: 11458 (systemd-timesyn)
     Status: "Initial synchronization to time server 125.229.162.223:123 (0.suse.pool.ntp.org)."
      Tasks: 2
     CGroup: /system.slice/systemd-timesyncd.service
             └─ 11458 /usr/lib/systemd/systemd-timesyncd

May 27 09:44:46 harvester-node-0 systemd[1]: Starting Network Time Synchronization...
May 27 09:44:47 harvester-node-0 systemd[1]: Started Network Time Synchronization.
May 27 09:44:52 harvester-node-0 systemd-timesyncd[11458]: Initial synchronization to time server 125.229.162.223:123 (0.suse.pool.ntp.org).
```

```bash
harvester-node-0:~ # timedatectl show-timesync
FallbackNTPServers=0.suse.pool.ntp.org 1.suse.pool.ntp.org 2.suse.pool.ntp.org 3.suse.pool.ntp.org
ServerName=0.suse.pool.ntp.org
ServerAddress=125.229.162.223
RootDistanceMaxUSec=5s
PollIntervalMinUSec=32s
PollIntervalMaxUSec=34min 8s
PollIntervalUSec=1min 4s
NTPMessage={ Leap=0, Version=4, Mode=4, Stratum=2, Precision=-23, RootDelay=8.697ms, RootDispersion=37.475ms, Reference=D8EF2304, OriginateTimestamp=Mon 2024-05-27 09:44:52 UTC, ReceiveTimestamp=Mon 2024-05-27 09:44:52 UTC, TransmitTimestamp=Mon 2024-05-27 09:44:52 UTC, DestinationTimestamp=Mon 2024-05-27 09:44:52 UTC, Ignored=no PacketCount=1, Jitter=0 }
Frequency=3391487
```

Reply:
Yeah, as you checked, we will use the Fallback NTP server, so I thought the empty should be fine.